apps/judicial-system/backend/src/app/modules/case/internalCase.controller.ts

```diff
@@ -68,6 +69,27 @@ export class InternalCaseController {
     return this.internalCaseService.archive()
   }
 
+  @Post('cases/indictments')
+  @ApiOkResponse({
+    type: Case,
+    isArray: true,
+    description: 'Gets all indictment cases',
+  })
+  getIndictmentCases(
+    @Body() internalCasesDto: InternalCasesDto,
+  ): Promise<Case[]> {
+    this.logger.debug('Getting all indictment cases')
+    const nationalId =
```

Comment:
We have a lib function called formatNationalId that does this as well, consider using that